applications/DeepSpeed-Chat/training/utils/data/raw_datasets.py

```diff
@@ -679,3 +722,33 @@ def get_prompt_and_rejected(self, sample):
             f"Warning: dataset {self.dataset_name} does not include rejected response."
         )
         return None
+
+
+class LocalParquetDataset(PromptRawDataset):
```

Comment:
Why do we need this class? This LocalParquetDataset class is never used in your PR. Please remove it if it's not necessary.